File: ios/RNMBX/RNMBXSymbolLayerViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(RNMBXSymbolLayer, RNMBXSymbolLayerViewManager, RCTViewManager)

// symbol layer props
RCT_EXPORT_VIEW_PROPERTY(sourceLayerID, NSString)

// standard layer props
#include "CommonLayerProperties.H"

@end

File: ios/RNMBX/RNMBXSymbolLayerViewManager.swift
@objc(RNMBXSymbolLayerViewManager)
class RNMBXSymbolLayerViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return false
    }
  
    override func view() -> UIView! {
      let layer = RNMBXSymbolLayer()
      layer.bridge = self.bridge
      return layer
    }
}
